content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,240 @@
+  customers;
+- :guilabel:`Discount Code`: Set codes which, when entered upon checkout, grants customers a
+  discount;
+- :guilabel:`Loyalty Cards`: Customers receive points when buying, which they can exchange for
+  rewards on their current or future order;
+- :guilabel:`Buy X Get Y`: Grant 1 credit for each item bought, then reward customers with Y item in
+  exchange of X credits.
+
+Configuration
+~~~~~~~~~~~~~
+
+When creating or modifying a program, the options available depend on the :guilabel:`Program Type`
+selected. Therefore, some programs may have additional options related to their type.
+
+- :guilabel:`Program Name`: Enter the name of the program, this is not visible by the customer;
+- :guilabel:`Program Type`: The :ref:`program type <pricing_management/program-types>` you want to
+  use;
+- :guilabel:`Currency`: The currency you wish to use for that program;
+- :guilabel:`Points Unit`: The unit you wish to use for the :guilabel:`Loyalty Cards`. You are free
+  to enter any name you want;
+- :guilabel:`Validity`: The date until which the program is valid;
+- :guilabel:`Limit Usage`: The limited number of times the program can be used during the
+  :guilabel:`Validity`;
+- :guilabel:`Company`: If you have multiple companies, the company in which the program is
+  available;
+- :guilabel:`Available On`: Check the app boxes where the program is available on;
+- :guilabel:`Website`: The website(s) on which the program is available. Leave the field to make it
+  available on *all* websites;
+- :guilabel:`Point of Sale`: The :abbr:`PoS (Point of Sale)` in which the program is available.
+  Leave empty to make it available on *all* :abbr:`PoS (Point of Sale)`.
+
+.. image:: price_management/price-programs.png
+   :align: center
+   :alt: Program options
+
+You can see the number of promotional items available per program in the upper-right corner, under
+the corresponding tab.
+
+.. image:: price_management/price-programs-items.png
+   :align: center
+   :alt: Program items tab
+
+Conditional rules
+*****************
+
+Under the :guilabel:`Rules & Rewards` tab, you can set the conditions and points value depending on
+the program type you selected. Click :guilabel:`Add` in the :guilabel:`Conditional rules` column to
+add **conditions**.
+
+- :guilabel:`Minimum Quantity`: The minimum quantity that must be bought to access the reward;
+- :guilabel:`Minimum Purchase`: The minimum amount (in currency) :guilabel:`tax Included` or
+  :guilabel:`tax Excluded` to access the reward. You can combine both quantity *and* amount;
+- :guilabel:`Products`: The product(s) to which the program applies;
+- :guilabel:`Categories`: The category of products to which the program applies. Select
+  :guilabel:`All` to apply to *all* categories;
+- :guilabel:`Product Tag:` Select a tag to apply the program to products containing that tag;
+- :guilabel:`Grant`: Set the number of points the customer can earn :guilabel:`per order`,
```

Comment:
Maybe mention it only applies to _Loyalty Cards_ and _Buy X Get Y_

Reply:
Added a note